docs/code.md.backup

```diff
@@ -0,0 +1,37 @@
+# Code description
+
+## Global architecture
+
+The project is structured based on Ionic 4's default `tabs` layout.
+All of the code resides in the `src` folder.
+
+Images are to be put in the `src/assets` folder.
+Global CSS changes are made in the `src/theme/variables.scss` file.
+
+**The `src/app` folder contains all of the Typescript and HTML files.**
+
+The `model` folder contains the model classes.
+Each class corresponding to a main feature of the app is placed into the same folder. For example, the `area` folder contains the pages, components and services related to that feature.
+The `tab1`, `tab2` and `tabs` folders are generated by Ionic's `tabs` default layout.
+
+## Area
+
+`AreaDbService` is a service class that accesses the Storage database. It reads and modifies the `area` and `type` keys.
+
+`AddAreaPage` is a page that calls a `FormAreaComponent` in its HTML. It relies on the `FormAreaComponent` to handle the actual information input, and displays additional information and the submit button.
+It uses the URL parameter `parentid` to guess where to put the new area.
+
+`FormAreaComponent` is a generic form that can be used to add or edit an area. All information about the area is transmitted through the `area` input directive.
+
+## Operations
+
+`OperationDbService` is a service class that accesses the Storage database. It reads and modifies the `operation`, `label`, `vegetable` and `category` keys.
+
+
+### Planning
+
+`PlanningComponent` is called from the `Tab1Page` HTML. 
+
+`PlanningComponent` handles all the informations about the operations. It converts the dates, stored in "YYYY-MM-DD" into a "Do MMMM YYYY" format, more readable, and in french, using the [momentjs](https://momentjs.com/docs/) library.
+
+To scroll into the current when the component is started, the [ngx virtual scroller](https://www.npmjs.com/package/ngx-virtual-scroller) is used. At the moment of development, Ionic virtual scroll appears to be buggy in Ionic4 and Angular virtual scroll slowed down the application.  
```

Comment:
This file appears to be a duplicate of code.md. Is it supposed to have been committed ?

Reply:
good question, i thought this file was always there but it appears not...